libs/ballot-interpreter-vx/jest.config.js

```diff
@@ -5,13 +5,13 @@ const shared = require('../../jest.config.shared');
  */
 module.exports = {
   ...shared,
-  collectCoverageFrom: [...shared.collectCoverageFrom, '!test/fixtures/**/*'],
+  collectCoverageFrom: [...shared.collectCoverageFrom],
   coverageThreshold: {
     global: {
-      statements: 91,
-      branches: 77,
-      lines: 92,
-      functions: 85,
+      statements: 0,
+      branches: 0,
+      lines: 0,
+      functions: 0,
```

Comment:
Are you leaving this at 0? If so, can you add a comment explaining why?

Reply:
I reset them in a later PR after changing the purpose of the library to be for BMD ballots.